### ISSUE TITLE
Clostest distance between Shape (LineString) and a Point

### ISSUE THREAD

**Issue by mst94** [locationtech/spatial4j#230]
Is it possible to get the distance to the closest part of a LineString to a Point?

In JTS, it is possible, but in Spatial4J I don't figure it out how do it exactely.
![2023-08-23 14_49_41-Window](https://github.com/locationtech/spatial4j/assets/53451581/55c4ad7c-20f1-4291-a284-0c6056a2bfcb)

**Comment by dsmiley** [comment]
I don't think so.  Spatial4j is more focused on predicates like intersection, contains, within.  There is point-to-point distance via `DistanceCalculator` but not point to line.

**Comment by mst94** [comment]
Okay, thanks for the answer!

Am Do., 24. Aug. 2023 um 07:11 Uhr schrieb David Smiley <
***@***.***>:
